src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -99,9 +100,30 @@ private static bool Contract_Update(ApplicationEngine engine)
 
             private static bool Contract_Destroy(ApplicationEngine engine)
             {
+                if (!engine.TryPop(out StackItem sendTo)) return false;
+
```

Comment:
It's a little strange with a `sendTo` parameter. I think it's better to payback in contract, and we already have a NEP5 demo contract.

Reply:
I think that it's easier for developers, specify where they want to send the funds during destroy, ethereum has this behavior and I think that it's a good one, we have a deterministic address, ethereum not, and you can deploy it again, but if in the future the deploy cost it's higher like it was in neo2, i think that it's a big workaround that should not be required.